### PR TITLE
Introduce T0 evaluation task

### PR DESCRIPTION
**Pull request by thomasw21**

### CODE REVIEW

File: bigscience/gins/task.py [hunk 1]
```diff
@@ -43,6 +43,134 @@ def full_lm(dataset, sequence_length, output_features):
...
+    "super_glue_rte_does_this_imply",
+    "super_glue_rte_guaranteed_true",
+    "super_glue_rte_justified_in_saying",
+    "super_glue_rte_must_be_true",
+    "super_glue_rte_should_assume",
+
+    # WSC
+    "super_glue_wsc.fixed_GPT_3_Style",
+    "super_glue_wsc.fixed_I_think_they_mean",
+    "super_glue_wsc.fixed_Who_or_what_is_are",
```
Comment (craffel) [inline bigscience/gins/task.py:130]: I think you are missing `super_glue_wsc.fixed_Who_is_are` - that one was included in our eval and plots. Did it not show up for you or something? If so, I wonder if it got removed.

File: bigscience/gins/task.py [hunk 2]
```diff
@@ -43,6 +43,134 @@ def full_lm(dataset, sequence_length, output_features):
...
+    "super_glue_wsc.fixed_replaced_with",
+    "super_glue_wsc.fixed_the_pronoun_refers_to",
+
+    # Winogrande
+    "winogrande_winogrande_xl_Replace",
+    "winogrande_winogrande_xl_does_underscore_refer_to",
+    "winogrande_winogrande_xl_fill_in_the_blank",
+    "winogrande_winogrande_xl_stand_for",
+    "winogrande_winogrande_xl_underscore_refer_to",
+    "winogrande_winogrande_debiased_Replace",
```
Comment (craffel) [inline bigscience/gins/task.py]: you can remove all the debiased ones

**Review by craffel — CHANGES_REQUESTED** [review]

File: bigscience/gins/task.py (hunk 1 above)
Comment (thomasw21) [reply bigscience/gins/task.py:130]: Ah I think this template was named like that prior to https://github.com/bigscience-workshop/promptsource/pull/487 Do you want me to revert back to a commit before when launching evaluation?

Comment (craffel) [reply bigscience/gins/task.py:130]: Looks good to me, thanks.

**Review by craffel — APPROVED** [review]

**Pull request merged**